### PR TITLE
feat(api): return more info about workflow template parsing and execution errors

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-engine-cli-tools#8014.0 ✘  
* Build minimal
  * cdsctl linux ✘  
  * Engine Build linux amd64 ✔ 
  * tools/debpacker ✔ 
  * tools/os-ansible-inventory ✔ 
  * Worker linux amd64 / 386 ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#8014.0 ✘  
* Stage 1
  * Engine Test with DB ✘

File: engine/api/workflowtemplate/execute.go [hunk 1]
```diff
@@ -38,85 +40,156 @@ func prepareParams(wt *sdk.WorkflowTemplate, r sdk.WorkflowTemplateRequest) inte
...
+	if instance != nil {
+		data = map[string]interface{}{
+			"id":     instance.ID,
+			"name":   instance.Request.WorkflowName,
+			"params": prepareParams(wt, instance.Request),
+		}
 	}
 
-	out, err := executeTemplate(string(v), data)
+	var parsingErrs []error
```
Comment (fsamin) [inline engine/api/workflowtemplate/execute.go]: we have a sdk.MultiError available, you have to check if it fits your need

Comment (richardlt) [reply engine/api/workflowtemplate/execute.go]: done

File: engine/api/workflowtemplate/execute.go [hunk 2]
```diff
@@ -38,85 +40,156 @@ func prepareParams(wt *sdk.WorkflowTemplate, r sdk.WorkflowTemplateRequest) inte
...
+	if instance != nil {
+		data = map[string]interface{}{
+			"id":     instance.ID,
+			"name":   instance.Request.WorkflowName,
+			"params": prepareParams(wt, instance.Request),
+		}
 	}
 
-	out, err := executeTemplate(string(v), data)
+	var mutliErr sdk.MultiError
```
Comment (fsamin) [inline engine/api/workflowtemplate/execute.go]:
```suggestion
	var multiErr sdk.MultiError
```

Comment (richardlt) [reply engine/api/workflowtemplate/execute.go]: done

**Pull request merged**